docs/conf.py

```diff
@@ -71,14 +71,11 @@
 gettext_compact = False
 
 html_theme = 'sphinx_rtd_theme'
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
A space after the `#`? Just being a little nitpick about the style p:

Reply:
👍